app/modules/kit-library/assets/js/hooks/use-kit-call-to-action-updated-url.js

```diff
@@ -0,0 +1,12 @@
+export default function useKitCallToActionUpdatedUrl( url, kitId, kitTitle ) {
+	if ( kitTitle ) {
+		const cleanTitle = kitTitle.replace( /\s/g, '-' ).replace( /[^\w-]/g, '' ).toLowerCase();
+		url += `&utm_term=${ cleanTitle }`;
+	}
+
+	if ( kitId ) {
+		url += `&utm_content=${ kitId }`;
+	}
+
+	return url;
+}
```

Comment:
You can't be sure that the URL will already have a query parameter already.
Use the [`URL()`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) and [`URLSearchParams()`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) objects

Reply:
Please check updated code